recipes/cutadapt/meta.yaml

```diff
@@ -1,32 +1,31 @@
 package:
   name: cutadapt
-  version: '1.14'
+  version: '1.15'
 
 about:
   home: https://cutadapt.readthedocs.io/
   license: MIT License
   summary: 'trim adapters from high-throughput sequencing reads'
 
 source:
-  fn: cutadapt-1.14.tar.gz
-  url: https://pypi.python.org/packages/16/e3/06b45eea35359833e7c6fac824b604f1551c2fc7ba0f2bd318d8dd883eb9/cutadapt-1.14.tar.gz
-  md5: 57403c3794aa940699df3032a5e9b208
+  fn: cutadapt-1.15.tar.gz
+  url: https://pypi.python.org/packages/5a/89/a041e0a108bef64b1347e9b7126d813e61f0ce028d3db830fb7b77aab1ab/cutadapt-1.15.tar.gz
+  md5: 50baff135228644e8c68e90e2bd4e953
 
 build:
   number: 0
-  script: $PYTHON setup.py install
+  script: pip install .
```

Comment:
Hi @marcelm! Why did you changed this here? Please use `$PYTHON setup.py install`.

Reply:
See http://www.python3statement.org/practicalities/ (fourth bullet point) and also https://youtu.be/2DkfPzWWC2Q?t=17m34s .

It probably doesn’t matter so much in a recipe, but I would like to stop using `setup.py` directly wherever I can.

Reply:
I'm aware of this, but we try to align with the conda-forge guys atm and the current recommendation is to still use setup.py afaik.